docs/outputs.rst

```diff
@@ -22,207 +21,330 @@ FMRIPrep generates three broad classes of outcomes:
-the derivative of RMS variance over voxels (or :abbr:`DVARS (defined in Power, et al. 2012)`), and
-both the original (``dvars``) and standardized (``std_dvars``) are provided;
-``framewise_displacement`` is a quantification of the estimated bulk-head motion;
-``trans_x``, ``trans_y``, ``trans_z``, ``rot_x``, ``rot_y``, ``rot_z`` are the 6 rigid-body
-motion-correction parameters estimated by fMRIPrep;
-if present, ``non_steady_state_outlier_XX`` columns indicate non-steady state volumes with a single
-``1`` value and ``0`` elsewhere (*i.e.*, there is one ``non_steady_state_outlier_XX`` column per
-outlier/volume);
-additional noise components are calculated using :abbr:`CompCor (Component Based Noise Correction)`,
-according to both the anatomical (``a_comp_cor_XX``) and temporal (``t_comp_cor_XX``) variants;
-and the motion-related components identified by
-:abbr:`ICA (independent components analysis)`-:abbr:`AROMA (Automatic Removal Of Motion Artifacts)`
-(if enabled) are indicated with ``aroma_motion_XX``.
+The :abbr:`BOLD (blood-oxygen level dependent)` signal measured with fMRI is a mixture of fluctuations
+of both neuronal and non-neuronal origin.
+Neuronal signals are measured indirectly as changes in the local concentration of oxygenated hemoglobin.
+Non-neuronal fluctuations in fMRI data may appear as a result of head motion, scanner noise,
+or physiological fluctuations (related to cardiac or respiratory effects).
+For a detailed review of the possible sources of noise in the BOLD signal, refer to [Greve2013]_.
+
+*Confounds* (or nuisance regressors) are variables representing fluctuations with a potential non-neuronal origin.
+Such non-neuronal fluctuations may drive spurious results in fMRI data analysis,
+including standard activation :abbr:`GLM (General Linear Model)` and functional connectivity analyses.
+It is possible to minimize confounding effects of non-neuronal signals by including them as nuisance regressors
+in the :abbr:`GLM (General Linear Model)` design matrix or regressing them out from
+the fMRI data - a procedure known as *denoising*.
+There is currently no consensus on an optimal denoising strategy in the fMRI community.
+Rather, different strategies have been proposed, which achieve different compromises between
+how much of the non-neuronal fluctuations are effectively removed, and how much of neuronal fluctuations
+are damaged in the process.
+The *fMRIPrep* pipeline generates a large array of possible confounds.
+Note, *fMRIPrep* does not perform any denoising itself and it is up to the user to perform this step.
+
+The most well established confounding variables in neuroimaging are the six head motion parameters
+(three rotations and three translations) - the common output of the head motion correction
+(also known as *realignment*) of popular fMRI preprocessing software
+such as SPM_ or FSL_.
+One of the biggest advantages of *fMRIPrep* is the automatic calculation of multiple potential confounding variables
```

Comment:
This feels redundant after `The *fMRIPrep* pipeline generates a large array of possible confounds.` Further, the sequence structure feels abrupt. What if we ended the preceding paragraph with "damaged in the process." and rewrote this one:

> The most well established confounding variables in neuroimaging are the six head motion parameters (three rotations and three translations) - the common output of the head motion correction (also known as *realignment*) of popular fMRI preprocessing software such as SPM_ or FSL_.
> Beyond the standard head motion parameters, the *fMRIPrep* pipeline generates a large array of possible confounds, which enable researchers to choose the most suitable denoising strategy for their analyses.
> *fMRIPrep* **does not** perform any denoising itself.


This is mutually exclusive with my previous suggestion.